_posts/2022-11-07-road-to-quarkus-3.adoc

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: 'Starting Quarkus 3'
+date: 2022-11-07
+tags: announcement
+synopsis: 'Quarkus 3 is coming with new features and a move to Jakarta named APIs.'
+author: maxandersen
+---
+
+Quarkus 3 will continue the tradition of propelling Java development forward by moving fast and bring a cutting-edge user experience with the latest technologies.
+
+Happy to have a first Quarkus 3 Alpha available for users to try out and extension writers to integrate with to get early feedback.
+
+## What is in it?
+
+The current planned list of highlights for Quarkus 3 are:
+
+  - *Hibernate 6* - faster, safer, better (Check out https://www.youtube.com/watch?v=pc6QIwx0EL0[this presentation] to see how awesome it is).
+  - *Jakarta EE 10* API and packages - following the latest evolution of the Java world
+  - *Micro profile 6* - the latest update of all the MP specifications continuing evolving alongside the state of the practice
+  - *HTTP/3* support (aka *Quic*) - the https://www.redhat.com/architect/http3[new HTTP version] fixing the head-of-line blocking problem, using UDP.
+  - *IO_URING* - the next generation of asynchronous IO support. This is a complete game changer if you are interested in performance and efficiency (response time, latency…)
+  - *Virtual threads* (aka Loom) and *Structured Concurrency Support* - based on the lessons learned from our https://www.redhat.com/architect/http3[initial integration], we are now looking into expanding and improving the support.
+  - *java.util.concurrent.Flow* - Time to evolve; Flow has been in the JDK since Java 9. Thus we want to move from Reactive Streams to java.util.concurrent.Flow.
```

Comment:
```suggestion
  - *java.util.concurrent.Flow* - Time to evolve; Flow has been in the JDK since Java 9. Thus we want to move from Reactive Streams to `java.util.concurrent.Flow`.
```